source/documentation/guides/joiners_movers_leavers.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Joiners Movers Leavers
+weight: 10
+last_reviewed_on: 2024-01-31
+review_in: 12 weeks
+---
+# <%= current_page.data.title %>
+
+The majority of our Joiners/Movers/Leavers processes can be carried out by using the opg-org-infra Terraform to assign permissions. Some tools like Jira or product-specific admin tools may require additional work. In general the JML process will be a combination of Delivery Manager and WebOps work at the delivery team level.
+
+##Joiners##
+
+New Developer or WebOps team members will need:
+
+- GitHub Access
```

Comment:
Is this org membership? that's managed by Ops Engineering

Reply:
I'll clarify as both org and team membership will be needed and have distinct processes.